Libplanet/Net/Protocols/KademliaProtocol.cs

```diff
@@ -684,22 +690,22 @@ private void ReceivePing(Ping ping)
             Address target,
             int depth,
             TimeSpan? timeout,
-            CancellationToken cancellationToken,
-            Address searchAddress)
+            CancellationToken cancellationToken)
         {
-            BoundPeer peerFound = null;
             List<BoundPeer> peers = found.Where(
                 peer => !peer.Address.Equals(_address)).ToList();
 
             if (peers.Count == 0)
             {
                 _logger.Debug("No any neighbor received.");
-                return peerFound;
+                return null;
```

Comment:
Would it be difficult to have a unit test for this case?

Reply:
I guess we have to make `ProcessFoundForSpecificAsync` method public (or internal) to make unit test for this case. Although, we can make test to cover this line for this case, but with no meaning 😥